.cirrus.yml

```diff
@@ -0,0 +1,64 @@
+freebsd_instance:
+  image: freebsd-11-2-release-amd64
+
+freebsd_build_task:
+
+  env:
+    matrix:
+      - configuration: Debug
+      - configuration: RelWithDebInfo
+
+  env:
+    matrix:
+      - arch: amd64
+      - arch: i386
+        jailName: j11i386
+        execPrefix: cbsd jexec jname=j11i386
+
+  env:
+    GH_API_TOKEN: ENCRYPTED[70e58bd8024a69b850f0dd50d5c86b1eeefbb5b4361b69c0086d16ccff7e66371486fe2d986f2ec6c2df26cf9a1a0561]
```

Comment:
@Snaipe, this token is for publishing release artifacts. Here are all the steps I used:

1. Allowed CirrusCI to access Criterion: https://github.com/marketplace/cirrus-ci
2. Created a 'personal access token' with just one scope checkbox checked: `public_repo` https://github.com/settings/tokens <br/><img src="https://user-images.githubusercontent.com/3840695/52527184-0d072280-2ccd-11e9-9520-848a691efb9f.png" alt="screenshot-1" width="42%" />
3. Logged on to CirrucCI using GH account and encrypt token using `Secured Variables` facility provided at the setting page: <br/> <img src="https://user-images.githubusercontent.com/3840695/52527210-9cacd100-2ccd-11e9-9d98-46a28f777cfa.png" width="42%" alt="screenshot-2" />